Jenkinsfile

```diff
@@ -516,7 +516,7 @@ pipeline {
                             ulimit -c unlimited
                             cd build
                             CXX=/opt/rocm/llvm/bin/clang++ cmake -DMIOPEN_USE_MLIR=On -DBUILD_DEV=On -DCMAKE_BUILD_TYPE=release -DMIOPEN_TEST_FLAGS='--verbose --disable-verification-cache' ..
-                            CTEST_PARALLEL_LEVEL=4 MIOPEN_LOG_LEVEL=5 make -j\$(nproc) check
+                            CTEST_PARALLEL_LEVEL=4 MIOPEN_LOG_LEVEL=5 make -j\$(nproc) test_conv_igemm_mlir
```

Comment:
WRT "what if this experimental stuff will break our CI?" -- if MLIR gets broken, then CI must fail, even if MLIR support is yet experimental. "Experimental" simply means that we do not guarantee the users anything, but in all other aspects this is identical to the non-experimental stuff. 

Obviously, MLIR-related PRs should not break MLIR. Likewise, all other PR should not break MLIR, otherwise their PRs should fail on CI.

/cc @shurale-nkn

Reply:
Addressed in 8fc3457